packages/documentation-site/patternfly-docs/content/design-guidelines/content/punctuation.md

```diff
@@ -3,95 +3,99 @@ id: Punctuation
 section: UX writing
 ---
 
-## Headings and titles
-Headings and titles can include punctuation, but should not end in punctuation. For example:
+## Usage 
 
-<div class="ws-content-table">
-
-|**Before**  | **After** |
-|------------|-----------|
-| Style, voice, and tone. | Style, voice, and tone |
-| Getting started with PatternFly! | Getting started with PatternFly |
+These best-practice recommendations cover some of the most common UI scenarios and use cases. For additional guidance, we also have recommendations for the use of specific [punctuation marks](#punctuation-marks).
 
-</div>
+### Buttons 
+Avoid using punctuation in button labels.
 
-The exception for this rule is a question mark, when it is contextually important. For example, in a confirmation dialog, it is important that users understand what action they are about to take. A valid heading may include a question mark, such as "Delete service account?". 
+Do not use punctuation to create icons on buttons (for example, do not use a plus sign "+"). Instead, refer to [PatternFly's icons page](/design-foundations/icons) for any icons you place on buttons.
 
-## Referring to text in the UI
-When referring to an element or text in the UI, use bold text. Don't use quotation marks or italics -- those should be reserved for quotes and emphasis, respectively. For example:
+### Headings and titles
+Headings and titles can include punctuation, but should not end in punctuation. 
```

Comment:
it's mentioned below the table in that section -- do you think I should move it up and/or further emphasize?

<img width="873" alt="image" src="https://github.com/patternfly/patternfly-org/assets/105813956/60cc8b53-8037-4956-8530-c6d1144220f1">

Reply:
Yes, I saw that. But when I looked at the text, it appeared that the exception was stated much later. But since it's actually pretty close (in the same section), then nope we don't need to mention it twice! That was my bad, I should've looked at the preview in addition to the raw text :D

Reply:
no worries - this preview link has been broken for me half the time anyways, so it's a gamble if it would've helped lol. ty for the review!